llvm/lib/Target/RISCV/RISCVSchedSiFive7.td

```diff
@@ -213,12 +213,12 @@ let SchedModel = SiFive7Model in {
 let BufferSize = 0 in {
 def SiFive7PipeA       : ProcResource<1>;
 def SiFive7PipeB       : ProcResource<1>;
+def SiFive7IDiv        : ProcResource<1> { let Super = SiFive7PipeB; } // Int Division
```

Comment:
I don't know why these need a Super class. Their usages already say SiFive7PipeB in the ResourceCycles explicitly.

Reply:
I didn't mean to keep the Super class. As @wangpc-pp points out that the description of the PR says it is removed. I will push a fixup.